apps/web/components/base/navigation-menu.tsx

```
@@ -84,14 +91,27 @@ const NavigationMenuList = React.forwardRef<
 ))
 NavigationMenuList.displayName = NavigationMenuPrimitive.List.displayName
 
+/**
+ * Individual navigation menu item.
+ */
 const NavigationMenuItem = NavigationMenuPrimitive.Item
 
+/**
+ * Styles for navigation menu trigger button.
+ */
+const navigationMenuTriggerStyle = cva(
```

Biome (1.9.4)

[error] Shouldn't redeclare 'navigationMenuTriggerStyle'. Consider to delete it or rename it.

'navigationMenuTriggerStyle' is defined here:


(lint/suspicious/noRedeclare)